.github/workflows/test_and_build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - uses: actions/checkout@v2
 
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
```

Comment:
what is this?

Reply:
v1 does not support python 3.5 anymore and checks fail. We either have to change the version matrix as `python-version: [3.6, 3.8]` or use `setup-python@v2`